src/templates/layout.html

```diff
@@ -7,6 +7,12 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
     <title>Outreach Portal</title>
+    <!-- active-inactive styles-->
+        <link data-require="bootstrap@*" data-semver="3.2.0" rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.css" />
```

Comment:
Please remove **Vlabs logo link redirecting to open-edx url** code from this branch.